pkgs/applications/version-management/git-and-tools/fast-export/default.nix

```
@@ -1,41 +1,42 @@
-{stdenv, fetchgit, mercurial, makeWrapper, subversion}:
+{stdenv, fetchgit, mercurial, makeWrapper}:
 
-with stdenv.lib;
-stdenv.mkDerivation {
-  name = "fast-export";
+stdenv.mkDerivation rec {
+  pname = "fast-export";
+  version = "v190107";
```

```suggestion
  version = "190107";
```
Version must start by a digit.